ci/release/update-version.sh

```diff
@@ -79,3 +79,12 @@ sed_runner "s/^version = .*/version = \"${NEXT_FULL_TAG_PEP440}\"/g" python/cusp
 
 # Dependency versions in dependencies.yaml
 sed_runner "/-cu[0-9]\{2\}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" dependencies.yaml
+
+# Versions in README.md
+sed_runner "s/cuspatial:[0-9]\+.[0-9]/cuspatial:${NEXT_SHORT_TAG}/g" README.md
+sed_runner "s/cuspatial=[0-9]\+.[0-9]/cuspatial=${NEXT_SHORT_TAG}/g" README.md
+sed_runner "s/cuspatial:[0-9]\+.[0-9]/notebooks:${NEXT_SHORT_TAG}/g" README.md
```

Comment:
This will replace `cuspatial:23.08` with `notebooks:23.10` (for example). I think this is a typo?

Reply:
?

```suggestion
sed_runner "s/notebooks:[0-9]\+.[0-9]/notebooks:${NEXT_SHORT_TAG}/g" README.md
```

Reply:
Yup that was the intention, that's what I get for a copy/paste without a critical eye. TY for that, would have been a mess. 

(This is needed because of the new docker pull commands where it's rapids/notebooks:version)